docs/source/noise/row_noise.rst

```diff
@@ -89,3 +89,50 @@ parameter:
 
 When applying :code:`omit_row` noise, each row of data is selected for omission
 independently with probability :code:`row_probability`.
+
+Duplicate with guardian
+-----------------------
+
+A known entity resolution challenge is children being reported multiple
```

Comment:
```suggestion
A known entity resolution challenge is people being reported multiple
```

I think "children" doesn't obviously include college students, and it is explained in more detail in the next sentence anyway.